legacy_packages/sdk/src/evm/zksync/temp-artifact/ZkSingletonFactory.ts

```diff
@@ -0,0 +1,7 @@
+export const zkSingletonFactoryArtifact = {
```

Comment:
do we need both bytecode and deployedBytecode here? if we do that's fine but if we don't then omitting one if we don't need it would save a lot of size